MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -875,12 +875,12 @@ public override List<string> GetLogFilePaths(string basePath)
                         logFiles.Add($"{basePath}.toc");
 
                     // Include .hash and .skeleton for all files in cuesheet
-                    CueSheet? cueSheet = new SabreTools.Serialization.Files.CueSheet().Deserialize($"{basePath}.cue");
-                    if (cueSheet != default && cueSheet.Files != null)
+                    var cueSheet = new SabreTools.Serialization.Files.CueSheet().Deserialize($"{basePath}.cue");
+                    if (cueSheet?.Files != null)
                     {
                         foreach (CueFile? file in cueSheet.Files)
                         {
-                            string trackPath = $"{Path.GetDirectoryName(basePath)}\\{Path.GetFileNameWithoutExtension(file?.FileName)}";
+                            string trackPath = $"{Path.Combine(Path.GetDirectoryName(basePath), Path.GetFileNameWithoutExtension(file?.FileName))}";
```

Comment:
Since you're doing a combine, you don't need it to be embedded in a string. The enclosing `$"{}"` can be removed.